h2o-persist-hdfs/build.gradle

```diff
@@ -9,7 +9,7 @@ configurations {
 dependencies {
     compile project(":h2o-core")
     compile('net.java.dev.jets3t:jets3t:0.9.0')
-    compile("org.apache.hadoop:hadoop-hdfs:$defaultHadoopClientVersion") {
+    compile("org.apache.hadoop:hadoop-hdfs-client:$defaultHadoopClientVersion") {
```

Comment:
I like this, lesser the dependencies, the better!

Reply:
Thanks @jakubhava. This will hopefully also help with the size of the artifact. Current size is > 145 MB, with this change we are on 125 MB. Limit for pypi is 130.